README.md

```diff
@@ -62,9 +67,23 @@ npm run test:e2e      # runs all E2E tests in headless mode
 npm run test:cy       # opens the Cypress app to run E2E tests visually
 ```
 
-## Project Structure
-
-Work in Progress, will finish when we solidify our project structure.
+## 📂 Project Structure
+
+There are four main folders in this project:
```

Comment:
🔧 Add the lib folder to this as well. It contains frontend components and most importantly, helpers/DTOs for the API routes!

Reply:
done